swmmio/tests/test_graphics.py

```diff
@@ -73,3 +76,106 @@ def test_change_crs():
     v2_test = pd.read_csv(StringIO(s), index_col=0, delim_whitespace=True, skiprows=[0])
     assert v2['X'].values == pytest.approx(v2_test['X'].values, rel=1e-3)
     assert v2['Y'].values == pytest.approx(v2_test['Y'].values, rel=1e-3)
+
+
+profile_selection_assert = \
+    {
+        'nodes':
+            [{'id_name': '9', 'rolling_x_pos': 0.0, 'invert_el': 1000.0},
+             {'id_name': '10', 'rolling_x_pos': 400.0, 'invert_el': 995.0},
+             {'id_name': '21', 'rolling_x_pos': 800.0, 'invert_el': 990.0},
+             {'id_name': '24', 'rolling_x_pos': 1300.0, 'invert_el': 984.0},
+             {'id_name': '17', 'rolling_x_pos': 1700.0, 'invert_el': 980.0},
+             {'id_name': '18', 'rolling_x_pos': 2100.0, 'invert_el': 975.0}],
+        'links':
+            [{'id_name': '1', 'rolling_x_pos': 200.0, 'midpoint_bottom': 997.5},
+             {'id_name': '6', 'rolling_x_pos': 600.0, 'midpoint_bottom': 993.0},
+             {'id_name': 'LOOP', 'rolling_x_pos': 1050.0, 'midpoint_bottom': 987.0},
+             {'id_name': '16', 'rolling_x_pos': 1500.0, 'midpoint_bottom': 982.0},
+             {'id_name': '10', 'rolling_x_pos': 1900.0, 'midpoint_bottom': 977.5}],
+        'path_selection':
+            [('9', '10', '1'),
+             ('10', '21', '6'),
+             ('21', '24', 'LOOP'),
+             ('24', '17', '16'),
+             ('17', '18', '10')]
+    }
+
+
+def test_profile():
```

Comment:
For some reason, this unit test fails on Apple M1. But it seems fine in the rest of the build matrix. I think we'll need to revisit this when we work on #167. This is okay for this PR.